Makefile

```diff
@@ -1,11 +1,17 @@
-.PHONY: test
+.PHONY: lint test 
 
 GO=CGO_ENABLED=0 GO111MODULE=on go
+ARCH=$(shell uname -m)
 
 tidy:
 	go mod tidy
 
-test:
+lint:
+	@which golangci-lint >/dev/null || echo "WARNING: go linter not installed. To install, run\n  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b \$$(go env GOPATH)/bin v1.42.1"
+	@if [ "z${ARCH}" = "zx86_64" ] && which golangci-lint >/dev/null ; then golangci-lint run --config .golangci.yml ; else echo "WARNING: Linting skipped (not on x86_64 or linter not installed)"; fi
+
+
+test: lint
```

Comment:
Restructure this as @bnevis-i did for this PR: https://github.com/edgexfoundry/go-mod-configuration/pull/57/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

Reply:
I have restructured the Makefile per your suggestion